content/blog/2017/2017-02-06-declarative-pipeline-ga.adoc

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Declarative Pipeline Syntax 1.0 is GA"
```

Comment:
I'm not sure the term "is GA" is hugely well known, and seems a bit out of place in the title — maybe more simply, "is here", or "released"…